Standards/CCSMB-1.md

```diff
@@ -0,0 +1,14 @@
+# CCSMB-1: Procedures for submitting a standard
+
+This RFC defines the protocol for defining and submitting a standard to the CCSMB.
+
+## File names and formatting
+
+Each RFC should be named `CCSMB-n.md`, where `n` is a number counting upwards.
+
+Each RFC should be a [GitHub-Flavored Markdown](https://github.github.com/gfm/) document with American English spelling.
```

Comment:
The way it has always been in CC is British English spelling.

Reply:
Ah, let's do that then.
```suggestion
Each RFC should be a [GitHub-Flavoured Markdown](https://github.github.com/gfm/) document with British English spelling.
```